components/ecosystem/TestTable.vue

```diff
@@ -8,21 +8,20 @@
       'Test Logs',
     ]"
   >
-    <cv-data-table-row
-      v-for="(row, rowIndex) in tableData"
-      :key="`${rowIndex}`"
-    >
-      <cv-data-table-cell
+    <bx-table-row v-for="(row, rowIndex) in tableData" :key="`${rowIndex}`">
+      <bx-table-cell
         v-for="({ component, styles, data, addTooltip }, elementIndex) in row"
         :key="`${elementIndex}`"
       >
         <span v-if="component === 'span'" :style="styles">
           {{ data }}
-          <cv-tooltip
+          <bx-tooltip-icon
             v-if="addTooltip"
-            :tip="testTypeTooltip[data]"
+            :body-text="testTypeTooltip[data as keyof typeof testTypeTooltip]"
```

Comment:
If you need to specify the type of `data`, I think you can do it when defining the `testType` on the `interface filteredDataItem`. This way you calculate that value only once (right now it's inside a loop, so you evaluate the same code with each turn of the loop

Reply:
Changed the type assertion with https://github.com/Qiskit/qiskit.org/pull/3001/commits/c01b98a37424d6875442f1d4893c89ae545a84e5.
The assertion being inside a loop might have no meaningful impact given that it would compile away on build time.